src/NzbDrone.Core/Download/ProcessDownloadDecisions.cs

```diff
@@ -36,26 +38,24 @@ public ProcessedDecisions ProcessDecisions(List<DownloadDecision> decisions)
             var prioritizedDecisions = _prioritizeDownloadDecision.PrioritizeDecisions(qualifiedReports);
             var grabbed = new List<DownloadDecision>();
             var pending = new List<DownloadDecision>();
+            var storedUsenet = new List<DownloadDecision>();
+            var storedTorrent = new List<DownloadDecision>();
 
             foreach (var report in prioritizedDecisions)
             {
                 var remoteEpisode = report.RemoteEpisode;
-
+                var downloadProtocol = report.RemoteEpisode.Release.DownloadProtocol;
                 var episodeIds = remoteEpisode.Episodes.Select(e => e.Id).ToList();
 
-                //Skip if already grabbed
-                if (grabbed.SelectMany(r => r.RemoteEpisode.Episodes)
-                                .Select(e => e.Id)
-                                .ToList()
-                                .Intersect(episodeIds)
-                                .Any())
+                // Skip if already grabbed
+                if (IsGrabbed(grabbed, report))
                 {
                     continue;
```

Comment:
In the future I'd like to get these added as 'Fallback' to the pending list, and either hidden or greyed out on the UI.